app/models/card.rb

```diff
@@ -1,6 +1,9 @@
 class Card < ApplicationRecord
   validates :original_text, :translated_text, :review_date, presence: true
   validate :same_words, on: :create
+  belongs_to :user, optional: true
+  scope :unreviewed, -> { where('review_date <= ?', Date.today) }
+  scope :get_random, -> { unreviewed.order('RANDOM()').first }
```

Comment:
do not use method names like a `get_something`. Use `.something`, `Card.random` here.